phf/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "phf"
 authors = ["Steven Fackler <sfackler@gmail.com>"]
-version = "0.4.6"
+version = "0.4.7"
```

Comment:
Could you revert the changes to the Cargo.tomls? I like to have the version bump in a commit on its own